.travis.yml

```diff
@@ -63,31 +73,29 @@ install:
   - wget https://github.com/google/protobuf/releases/download/v3.6.0/protoc-3.6.0-linux-x86_64.zip -O /travis-install/protoc.zip
   - sudo unzip /travis-install/protoc.zip -d /usr
 script:
+  - ./lint.sh
```

Comment:
Moved invoking the linter earlier on so that we can detect build failures sooner (i.e. it's kinda annoying to go through the hassle of building the SageMaker docker container only for linter failures to fail the build)